### PR TITLE
Changed Things

### PR DESCRIPTION
**Pull request by williamrockowl**
Changed the css file name to something more simple, as well as I added more content to the dropdown nav bar.

### CODE REVIEW

File: William_Saulon.htm [hunk 1]
```diff
@@ -77,20 +131,18 @@ <h1>A History Of Where I Lived</h1>
...
-    </div>
+<hr>
 
-    <div id="socialMedia">
-        <a href="http://www.instagram.com/willfrombuffalo" class="btn btn-default" target="_blank">
-            <img src="Image\instagram.png" alt="Instagram"></a>
+<h1>LINKS:</h1>
+    <div class="linkstosites">
+        <p>
+        <a href="https://sites.google.com/site/buffalolittlelamb8/" target=" blank">- CLICK HERE TO GO TO MY FAVORITE RESTAURANT IN BUFFALO, NY</a> </p>
```
Comment (billythakidd04) [inline William_Saulon.htm:139]: i think you meant `target="_blank"`

File: William_Saulon.htm [hunk 2]
```diff
@@ -77,20 +131,18 @@ <h1>A History Of Where I Lived</h1>
...
+            <p>
 
+        <a href="secondPage.php" target=" blank">- CLICK HERE TO GO TO MADLIBS!</a>
+    </p>
     </div>
+<hr>
```
Comment (billythakidd04) [inline William_Saulon.htm:145]: close tag

File: cssFolder/myStyles.css [hunk 3]
```diff
@@ -0,0 +1,242 @@
...
+    color: black;
+    padding: 12px 16px;
+    text-decoration: none;
+    font-size: 18px;
+    font-weight: bold;
+}
+
+.links-hidden {
+    display: block;
+}
```
Comment (billythakidd04) [inline cssFolder/myStyles.css:149]: you have `inline-block` in the section above, why `block` here?

**Review by billythakidd04 — CHANGES_REQUESTED** [review]
Few small changes but its getting there

File: William_Saulon.htm (hunk 2 above)
Comment (williamrockowl) [reply William_Saulon.htm:145]: using the hr tag as a break, closing it makes it dissapear.

File: William_Saulon.htm [hunk 4]
```diff
@@ -3,57 +3,111 @@
...
+            Web Design, CyberSecurity, Game Development, and possibly work for a big name company like <mark
+                class="blue">FaceBook</mark>
+            or <mark class="green">Google</mark>. Eventually, I would also want to go into the medical aspect of code
+            that would help
+            hospitals and doctors create new technology that can potentially save lives that we aren't able to save now.
         </p>
-        <p> I believe that for a good
-            <mark class="purple">web developer</mark>, you have to have that creative quality to be able to make your
-            website stand out.
+        <hr>
```
[the remark this answers is not in the document]
Comment (williamrockowl) [reply William_Saulon.htm:58]: using the hr tag as a break, closing it makes it dissapear.

[the remark this answers is not in the document]
Comment (billythakidd04) [reply William_Saulon.htm:55]: That's not how that works. You're doing some thing else wrong then. Close the tag, If it doesn't work send me a screenshot

File: William_Saulon.htm [hunk 5]
```diff
@@ -3,57 +3,111 @@
...
-        </p>
-        <p> One of my
-            <mark class="green">goals</mark> is to become a part-time developer before I graduate.
-        </p>
-        <p> I am a current
-            <mark class="yellow">computer science </mark>student and I know a bit of a couple different programming
-            languages.
+        <hr>
+        <div id=Image>
+            <img src="Image\IMG_0908.jpg" alt="Hiking Trip" style="width:300px;height:400px;">
```
[the remark this answers is not in the document]
Comment (billythakidd04) [reply William_Saulon.htm:38]: This still needs to be fixed

**Review by billythakidd04 — CHANGES_REQUESTED** [review]
Still missing some changes, especially the image paths. If the `<hr/>` tag gives you trouble, let me know I'll help.

**Review by billythakidd04 — APPROVED** [review]

**Pull request merged**